### ISSUE TITLE
is_network_master and NETWORK_MODE_INHERIT

### ISSUE THREAD

**Issue by Elinvention** [godotengine/godot#6475]
**Operating system or device - Godot version:**

Debian testing, Godot 2.2

**Issue description** (what happened, and what was expected):

```
NodeA            <-- master
|____> NodeB     <---inherit
```

NodeB.is_network_master() returns false. I think it should return true.

**Steps to reproduce:**

This code fails:

```
set_network_mode(NETWORK_MODE_MASTER)
assert(get_node("son").is_network_master())
```

**Comment by Elinvention** [comment]
That code works after a call to `create_server`, but fails after a call to `create_client` or when none of them were called.

**Comment by Elinvention** [comment]
This is fixed in 3.0.
